thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -607,7 +618,13 @@ private void decodeAndInvoke(
             return;
         }
 
-        invoke(ctx, serializationFormat, seqId, f, decodedReq, httpRes);
+        if (useBlockingTaskExecutor) {
+            ctx.blockingTaskExecutor().execute(
+                    () -> invoke(ctx, serializationFormat, seqId, f, decodedReq, httpRes)
+            );
```

Comment:
we think `invoke()` is only thing that we should wrap with `ctx.blockingTaskExecutor()`.

If not, please share to us~! 🙇